docker-compose.localstack-local.yaml

```diff
@@ -3,7 +3,7 @@ version: "3.8"
 services:
   localstack:
     container_name: "localstack-suspension-service"
-    image: localstack/localstack
+    image: localstack/localstack:2.3.2
```

Comment:
Weird one this one, I wouldn't push this up. I think this requires a spike, so the pipeline seems to pass with `localstack/localstack` - wouldn't want to risk shaking the pipeline before we can roll to pre-prod. Definitely tech debt again, I know it makes zero sense - me and Joe were baffled! Need to look at this after pen.